package.json

```diff
@@ -15,6 +15,7 @@
     "build:sniffer": "cd ./traffic-sniffer && npm run build",
     "build": "concurrently --kill-others \"cd ./traffic-sniffer && npm run dev\"  \"cd ./traffic-dashboard && npm run build\"",
     "dev": "concurrently --kill-others \"cd ./traffic-sniffer && npm run dev\"  \"cd ./traffic-dashboard && npm run dev\"",
+    "dev-server": "npm run dev && nodemon dummy-server.js && npm run dev",
```

Comment:
Remove the second npm run dev.
And consider using concurrently.

Reply:
consider changing the command to demo?

Reply:
Resolved